pkg/lint/support/message_test.go

```diff
@@ -45,6 +44,7 @@ func TestRunLinterRule(t *testing.T) {
 		{-1, errLint, 4, false, ErrorSev},
 	}
 
+	var linter = Linter{}
```

Comment:
I created https://github.com/helm/helm/pull/31024 to focus this PR on changing the test execution only